### PR TITLE
make the section describing wiring up the hit signal slightly more verbose

### PR DESCRIPTION
**Pull request by dprime**
As a new user I was following this tutorial and it wasn't clear to me that I had to have the instanced player selected and not the player scene itself. I got some help in discord and figured out what I needed to do and so I thought I'd push a tweak to the tutorial to hopefully make it clearer for the next person.

### CODE REVIEW

**Comment by Vennnot** [comment]
I think this can be merged

File: getting_started/first_2d_game/05.the_main_game_scene.rst [hunk 1]
```diff
@@ -202,17 +202,19 @@ You can assign this property's value in two ways:
...
 double-click the ``hit`` signal in the list (or right-click it and select
 "Connect..."). This will open the signal connection dialog. We want to make a
 new function named ``game_over``, which will handle what needs to happen when a
 game ends. Type "game_over" in the "Receiver Method" box at the bottom of the
-signal connection dialog and click "Connect". Add the following code to the new
-function, as well as a ``new_game`` function that will set everything up for a
-new game:
+signal connection dialog and click "Connect". You are aiming to have the ``hit``` signal
+emitted from ``Player`` and handled in the ``Master`` script. Add the following code
+to the new function, as well as a ``new_game`` function that will set
```
Comment (mhilbrunner) [inline getting_started/first_2d_game/05.the_main_game_scene.rst:216]:
```suggestion
signal connection dialog and click "Connect". You are aiming to have the ``hit`` signal
emitted from ``Player`` and handled in the ``Main`` script. Add the following code
to the new function, as well as a ``new_game`` function that will set
```

**Comment by mhilbrunner** [comment]
> I think this can be merged

Then I would caution that we should spend bit more time and review PRs carefully :) Just bumping things isn't helpful and only adds noise. This one had a syntax error and also referred to a non-existing `Master` script (I assume it should be `Main`) in what amounts to a three line change.

**Comment by mhilbrunner** [comment]
Merged. Thanks for contributing, sorry this has been sitting for so many moons, but alas, the backlog :)

**Pull request merged**